docs/source/installation/index.rst

```diff
@@ -12,4 +12,4 @@ Installation guides
     Development <development.rst>
     Plugins <plugins.rst>
     Translations <translations.rst>
-    LaTeX <latex.rst>
+    PDF Generation <pdf_generation.rst>
```

Comment:
I think we may reference the /latex/ URL in the forum etc. So please add a redirect in conf.py (there are already some at the end of the file)